### PR TITLE
fix: unit tests execution

### LINKED ISSUE #7560: [Analytics Hub] Fix unit tests in Analytics

### PR DESCRIPTION
**Pull request by wzieba**
Closes: #7560

### Description

This PR fixes broken unit tests in `feature/analytics` branch.

### Testing instructions

None - CI check is sufficient.

- [ ] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WooCommerce/src/test/kotlin/com/woocommerce/android/ui/analytics/AnalyticsViewModelTest.kt [hunk 1]
```diff
@@ -453,8 +453,10 @@ class AnalyticsViewModelTest : BaseUnitTest() {
         val getShowIndicatorStatesJob = launch { sut.state.toList(states) }
         sut.onRefreshRequested()
 
-        assertEquals(2, states.filter { it.refreshIndicator is NotShowIndicator }.size)
-        assertEquals(2, states.filter { it.refreshIndicator is RefreshIndicator.ShowIndicator }.size)
```
Comment (wzieba) [inline WooCommerce/src/test/kotlin/com/woocommerce/android/ui/analytics/AnalyticsViewModelTest.kt:457]: I don't know why there was `2` in this test before. The test should check if
1. There's no indicator at the beginning
2. There's is indicator later

which the new `assertThat` asserts.

File: WooCommerce/src/test/kotlin/com/woocommerce/android/ui/analytics/AnalyticsRepositoryTest.kt [hunk 2]
```diff
@@ -793,16 +797,23 @@ class AnalyticsRepositoryTest : BaseUnitTest() {
...
-        whenever(toPerformerProductModel.product).thenReturn(product)
-        return mutableListOf(toPerformerProductModel)
+
+        val productEntity = TopPerformerProductEntity(
+            siteId = 0,
+            granularity = "",
+            productId = product.remoteProductId,
+            name = product.name,
+            imageUrl = product.getFirstImageUrl(),
+            quantity = TEN_VALUE.toInt(),
```
Comment (wzieba) [inline WooCommerce/src/test/kotlin/com/woocommerce/android/ui/analytics/AnalyticsRepositoryTest.kt:811]: I'd like to point out that we should remove constants like `TEN_VALUE` or `ONE_HUNDRED_DECREASE`. It's the same "magic number" problem as having raw "10.0" here.

Also we should remove `ZERO_VALUE` as it's not a magic number and brings no value.

**Review by wzieba — COMMENTED** [review]
Hint: use the `hide whitespaces` option in this review, as indentation has been updated (and uncheck it later).

image

**Review by atorresveiga — APPROVED** [review]
LGTM! :shipit:

**Pull request merged**